src/Core/src/Platform/iOS/MauiLabel.cs

```diff
@@ -66,24 +66,6 @@ RectangleF AlignVertical(RectangleF rect)
 			return rect;
 		}
 
-		public override void InvalidateIntrinsicContentSize()
-		{
-			base.InvalidateIntrinsicContentSize();
-
-			if (Frame.Width == 0 && Frame.Height == 0)
-			{
-				// The Label hasn't actually been laid out on screen yet; no reason to request a layout
-				return;
-			}
-
-			if (!Frame.Size.IsCloseTo(AddInsets(IntrinsicContentSize), (nfloat)0.001))
-			{
-				// The text or its attributes have changed enough that the size no longer matches the set Frame. It's possible
-				// that the Label needs to be laid out again at a different size, so we request that the parent do so. 
-				Superview?.SetNeedsLayout();
-			}
-		}
```

Comment:
I have not seen any real cases where this call should/does/needs to invalidate the layout. We are already calling invalidate in the Label class when we change the properties (Text, Font, CharSpacing) so this is redundant.

Also, comparing `Frame` to `IntrinsicContentSize` is ALWAYS incorrect. Assume you have a label with text "Hello" and the width of that text is say 100, then you set the label width of 200. The Frame now has a width of 200, but IntrinsicContentSize has a width of 100... This means it will ALWAYS trigger a layout.